pkg/api/types.go

```diff
@@ -2262,6 +2262,11 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+const (
+	// Namespace prefix for opaque counted resources (alpha).
+	ResourceOpaqueIntPrefix = "opaque-int.alpha.kubernetes.io/"
```

Comment:
Related question, have been seeing comments lately concerned with proliferation of "alpha-prefixed" things in the API and confusion it might cause. Is there an official direction for what to do here?